core/types/rollup_cost_test.go

```diff
@@ -0,0 +1,91 @@
+package types
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/params"
+	"github.com/stretchr/testify/require"
+)
+
+func TestRollupGasData(t *testing.T) {
+	for i := 0; i < 100; i++ {
```

Comment:
Why is this in a loop? To ensure it is deterministic?

Reply:
whoops, the earlier test generated random data over 100 iterations... I removed the random data and substituted an actual test transaction but forgot to remove this loop. . fixed.